ci/supportedBranches.js

```diff
@@ -13,6 +13,16 @@ const typeConfig = {
   nixpkgs: ['channel'],
 }
 
+// "order" ranks the development branches by how likely they are the "right" branch when
+// they are an equally well fit as a target branch for ci/github-script/prepare.js.
```

Comment:
I think "equally well" should be "equally good. Here's one possible alternative phrasing:

```suggestion
// they are an otherwise equally good fit according to ci/github-script/prepare.js.
```

---

Maybe the prior line would also be clearer if it talked about the "intended target" or "intended base" instead of "right"?

Either is fine though, and the current wording is consistent with the variable name in `prepare.js`.